### PR TITLE
Makes tooltips good and creates a base for adding more.

### PR DESCRIPTION
**Pull request by SandPoot**
## About The Pull Request
Title, this pr will make it so tooltips can be added onto everything if set-up, by default it already adds it to humans and robots, it also solves the problem of trying to drag something but tooltips being on your way.

Example of moving something and not being blocked by tooltips:
i99l2DWgbs
Example of human with tooltips:
image
Example of borg with tooltips:
image
image
Example of something var-edited to have tooltips enabled, using default examine:
image

## Why It's Good For The Game
It is very good.

## Changelog
:cl:
tweak: Tooltips immediately go away on clicking, so you can drag stuff where the tooltip used to be.
fix: Fixed Cyborg examines adding an extra weird line.
code: Everything can be set to have tooltips, and even coded to have neat tooltips.
add: Makes it so humans and borgs already have tooltips.
/:cl:

### CODE REVIEW

File: code/modules/tooltip/tooltip.dm [hunk 1]
```diff
@@ -122,4 +122,27 @@ Notes:
 		if(user.client && user.client.tooltips)
 			user.client.tooltips.hide()
 
-
+/atom/movable
```
Comment (LetterN) [inline code/modules/tooltip/tooltip.dm]: move this var to atom movable

Comment (SandPoot) [reply code/modules/tooltip/tooltip.dm]: what file?

**Comment by SandPoot** [comment]
image

**Comment by silicons** [comment]
:heart:

**Pull request merged**